### ISSUE TITLE
md-tabs should respect the first auto-focus element in the md-tab

### ISSUE THREAD

**Issue by doug65536** [angular/material#5003]
When activating a tab, the tab should be checked for an element with an auto-focus attribute, and if there is one it should set focus to it.

**Comment by doug65536** [comment]
Oops, I meant [autofocus](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input#attr-autofocus), not `auto-focus`.

**Comment by marcysutton** [comment]
Do you mean the tab, or the tab content? There is focus behavior already set up on tabs so that keyboard and screen reader users can navigate them intuitively. What is your use case for sending focus to a different element?

**Comment by doug65536** [comment]
Yeah, you make a good point. The keyboard interface is fantastic. Almost all of the behaviour is exactly what I want. What I don't like is, if you select a tab that contains form controls (with enter, or by clicking the tab label), the user has to set focus into the tab content to use form controls (with tab press or click).

Is there a way for me to set focus to an input in that scenario?
